core/acl/common.go

```diff
@@ -9,6 +9,7 @@ const (
 
 // acl chaincode functions
 const (
+	FnAddUser                       = "addUser"
```

Comment:
why is this added here if it is only used in tests

Reply:
@pfi79 almost all functions listed in this file used only in tests

Reply:
Your first comment was about location of this constant, not about the function itself.
I agree that different way to create new users should be used in mocks.

Reply:
The other functions are necessary for the foundation to function

addUser is not needed here